src/ApparentHorizons/ComputeHorizonVolumeQuantities.hpp

```diff
@@ -68,12 +68,6 @@ struct ComputeHorizonVolumeQuantities
       const InverseJacobian<DataVector, 3, Frame::ElementLogical, TargetFrame>&
           inverse_jacobian);
 
-  using allowed_src_tags =
-      tmpl::list<gr::Tags::SpacetimeMetric<3, Frame::Inertial>,
-                 GeneralizedHarmonic::Tags::Pi<3, Frame::Inertial>,
-                 GeneralizedHarmonic::Tags::Phi<3, Frame::Inertial>,
-                 Tags::deriv<GeneralizedHarmonic::Tags::Phi<3, Frame::Inertial>,
-                             tmpl::size_t<3>, Frame::Inertial>>;
```

Comment:
I'm a bit confused by this, but that's probably because I don't know CCE that well. What tags are you using in CCE that aren't in this list? From a quick glance it seems like CCE only needs the metric, pi, and phi (as you removed the time derivatives).